redbot/core/modlog.py

```diff
@@ -1013,18 +1013,23 @@ async def create_case(
 
     Raises
     ------
+    ValueError
+        If the action type is not a valid action type.
+    RuntimeError
+        If user is the bot itself
     TypeError
         If ``channel`` is of type `discord.PartialMessageable`.
     """
     case_type = await get_casetype(action_type, guild)
     if case_type is None:
-        return
+        raise ValueError("That is not a valid action type.")
```

Comment:
Instead of "that", put the action type that was entered.
```suggestion
        raise ValueError(f"{action_type} is not a valid action type.")
```